packages/grafana-ui/src/components/Monaco/ReactMonacoEditorLazy.tsx

```diff
@@ -11,13 +16,14 @@ import type { ReactMonacoEditorProps } from './types';
  * @internal
  * Experimental export
  **/
-export const ReactMonacoEditorLazy = (props: ReactMonacoEditorProps) => {
+const MonacoEditorLazy = (props: ReactMonacoEditorProps) => {
+  const styles = useStyles2(getStyles);
   const { loading, error, dependency } = useAsyncDependency(
     import(/* webpackChunkName: "react-monaco-editor" */ './ReactMonacoEditor')
   );
 
   if (loading) {
-    return <LoadingPlaceholder text={''} />;
+    return <LoadingPlaceholder text={''} className={styles.container} />;
```

Comment:
What do you think about adding some text like "Loading editor" along with the spinner. I feel that it looks kind of lonely without it.

![imagen](https://github.com/grafana/grafana/assets/1069378/c252a574-8f6c-4491-ad80-cd0c56a0a76f)

Reply:
Yes, good suggestion!

https://github.com/grafana/grafana/assets/5732000/a47e6363-2d52-451c-8e16-e6eaec6b36ec

Implemented in [009e08c](https://github.com/grafana/grafana/pull/88102/commits/009e08c946988b5d023def0907aa392181ae3f0e). I'm very open to this and other ideas about the placeholder contents.

Reply:
Just a nit: Wouldn't it be nice to have loading spinner before the text?

Reply:
@itsmylife perhaps, but that is governed by the [`<LoadingPlaceholder />`](https://developers.grafana.com/ui/latest/index.html?path=/story/general-loadingplaceholder--basic).